react/GlovoAdmin.tsx

```diff
@@ -0,0 +1,814 @@
+  }
+
+  const deleteAffiliation = () => {
+    const tempDataDelete = [...settingsState.affiliation]
+
+    const newData = tempDataDelete.filter((item) => item.id !== itemDelete)
+
+    const settingsGlovo = bodySettings(newData)
+
+    saveData(settingsGlovo)
+  }
+
+  const customSchema = {
+    properties: {
+      nameAffiliation: {
+        title: formatIOMessage({
+          id: messageUI.affiliationName.id,
+          intl,
+        }).toString(),
+      },
+      salesChannel: {
+        title: formatIOMessage({
+          id: messageUI.salesChannel.id,
+          intl,
+        }).toString(),
+      },
+      pickupPoints: {
+        title: formatIOMessage({
+          id: messageUI.pickupPoints.id,
+          intl,
+        }).toString(),
+      },
+      actions: {
+        title: formatIOMessage({
+          id: messageUI.actions.id,
+          intl,
+        }).toString(),
+        cellRenderer: (e: TableItem) => cellComponent(e),
+      },
+    },
+  }
+
+  const editData = (itemEdit: AffiliationType) => {
+    const tempData = [...settingsState.affiliation]
+
+    tempData.map((item) => {
+      if (item.id === itemEdit.id) {
+        item.nameAffiliation = itemEdit.nameAffiliation
+        item.idAffiliation = itemEdit.idAffiliation
+        item.salesChannel = itemEdit.salesChannel
+        item.pickupPoints = itemEdit.pickupPoints
+        item.glovoId = itemEdit.glovoId
+      }
+
+      return item
+    })
+
+    const settingsGlovo = bodySettings(tempData)
+
+    saveData(settingsGlovo)
+  }
+
+  return (
+    <Layout fullWidth pageHeader={<PageHeader title="Glovo Settings" />}>
+      <Modal
+        isOpen={isOpenModal}
+        changeIsOpen={changeIsOpen}
+        type={typeModal}
+        saveData={getDataModal}
+        editData={editData}
+        affiliation={dataEdit}
+      />
+      <div className={styles.tableDesktop}>
+        <PageBlock>
+          <div style={{ height: '500px' }}>
+            <img src={iconGlovo} style={{ width: '80px' }} alt="" />
+            <div style={{ float: 'right' }} className="mb5">
+              <ButtonWithIcon
+                icon={add}
+                variation={NameFields.SECONDARY}
+                onClick={() => {
+                  setIsOpenModal(true)
+                  setFromMobile(false)
+                  setTypeModal(NameFields.TYPENEW)
+                }}
+              >
+                {formatIOMessage({
+                  id: messageUI.addMore.id,
+                  intl,
+                }).toString()}
+              </ButtonWithIcon>
+            </div>
+            <ModalDialog
+              centered
+              confirmation={{
+                onClick: () => deleteAffiliation(),
+                label: formatIOMessage({
+                  id: messageUI.dialogConfirmation.id,
+                  intl,
+                }).toString(),
+                isDangerous: true,
+              }}
+              cancelation={{
+                onClick: () => {},
```

Comment:
The `cancel` button on the Modal is not working. 
I'm guessing it is because we're not passing the `setAlertDialog(false)` here.